sos/policies/distros/redhat.py

```diff
@@ -380,17 +425,18 @@ def upload_archive(self, archive):
         """
         try:
             if self.upload_url and self.upload_url.startswith(RH_API_HOST) and\
-              (not self.get_upload_user() or not self.get_upload_password()):
+                    (not self.get_upload_user() or
+                     not self.get_upload_password()):
```

Comment:
I guess this is the culprit for the current problem: here we stopped checking for case ID (`get_upload_user` had that check) - but we still check user / password. Maybe:

                if self.upload_url and self.upload_url.startswith(RH_API_HOST) and\
                    not self.case_id:

?

Reply:
Let me check that - I actually solved it via upload_sftp directly, but this may be quicker

Reply:
Indeed, this seems to be solved well.